build.rs

```diff
@@ -9,7 +9,10 @@ fn main() -> Result<()> {
     tonic_build::configure()
         .build_server(true)
         .build_client(true)
-        .type_attribute(".", "#[derive(serde_derive::Serialize)]")
+        .type_attribute(
+            ".",
+            "#[derive(serde_derive::Serialize, serde_derive::Deserialize)]",
```

Comment:
this may break downstream users of the crate like api-rs (not tried yet). You don't need this derive for what this PR intended right?

Reply:
Yeah I don't, thought that it would be a pretty simple change but if it's not I'll get rid of it

Reply:
thanks yeah.. just an unknown so would need some testing with a few key downstream crates